source/views/sis/course-search/search.js

```diff
@@ -88,10 +100,9 @@ function executeSearch(args: {
 
 	const results = filteredCourses.filter(
 		course =>
-			fuzzysearch(query, course.name.toLowerCase()) ||
-			fuzzysearch(query, (course.title || '').toLowerCase()) ||
```

Comment:
Did you mean to remove the Title search here?

Reply:
Oops! Nope!